Formula/switch-lan-play.rb

```diff
@@ -0,0 +1,19 @@
+class SwitchLanPlay < Formula
+  desc "Make you and your friends play games like in a LAN"
+  homepage "https://github.com/spacemeowx2/switch-lan-play"
+  url "https://github.com/spacemeowx2/switch-lan-play/archive/a3ecfe6ef19c8496ebf8239a8eb15184cf9ee04d.tar.gz"
```

Comment:
Please use the release  tarball for 0.0.7 instead and you can delete the version line